actions/slot_match.py

```diff
@@ -58,3 +58,26 @@ def resolve_slot_match(
                 return resolved_child
 
     return {}
+
+
+def suggest_using_slot_match(
```

Comment:
I think it'd be good for us to return more than one value from the resolve_slot_match function. To refine how the user files down, but more importantly to order them. With this one, a lot of times it chose a weird service for what I said (since it was the first one it got to).

Reply:
We could have an additional param like "stop_on_first_match" and set it to True.